nestkernel/event.h

```diff
@@ -555,7 +555,7 @@ class DataLoggingRequest : public Event
   DataLoggingRequest();
 
   /** Create event for given time stamp and vector of recordables. */
```

Comment:
Could you extend this comment to explain the different parameters? That becomes more important since we have two `Time` arguments now.